sample-aws-ha-yamls/aws-ha-statefulset.yaml

```diff
@@ -103,6 +103,8 @@ spec:
                   key: nexus-admin-password
             - name: NEXUS_SECURITY_RANDOMPASSWORD
               value: "false"
+            - name: CLUSTERED_ZERO_DOWNTIME_ENABLED_ENV
```

Comment:
I just noticed, this shouldn't have an `_ENV` suffix

Reply:
Updated names, thanks!